[Docs] use "helm pull" instead of "git clone" to download helm charts faster (#197)
hwameistor/hwameistor: docs/docs/quick_start/install/storage_pool.md

## Set up storage pool during deployment

> A storage pool can be configured during HwameiStor deployment by setting the same `storageNodes` parameter:

Reviewer: ```suggestion
A storage pool can be configured during the HwameiStor deployment if you set the same `storageNodes` parameter:
```

> for example:

Reviewer: ```suggestion
For example:
```
